api-reference/autonomi-client/data_types/graph_entry.md

```diff
@@ -0,0 +1,77 @@
+## GraphEntry
+
+GraphEntry is a native data type in the Autonomi Network:  
+
+* **key-addressed:** Derived from a BLS public key
+* **Size:** 32 bytes of content per entry
+* **Immutable:** Once created, it cannot be changed  
+* **Directed Graph:** Can reference parent entries and descendant entries (with 32 bytes of metadata per descendant)  
+* **Signed:** Owned and signed by a key for cryptographic verification
+
+### Client Methods
+
+- **graph_entry_get**  
+  Retrieves a GraphEntry from the network by its address.
+- **graph_entry_put**  
+  Uploads a GraphEntry to the network with payment handling.
+- **graph_entry_cost**  
+  Estimates the storage cost for a GraphEntry.
+
+### Example
+
+```rust
```

Comment:
Have we removed the tabs for rust and python here? I think it's best to keep them, but follow up PR is cool